src/ledger/LedgerTxnImpl.h

```diff
@@ -310,14 +310,14 @@ class LedgerTxn::Impl
     //
     // We say an offer-descriptor A is worse than an offer-descriptor B if
     //     A.price > B.price || (A.price == B.price && A.offerID > B.offerID)
-    // We write this as A > B, and write !(A > B) as B <= A to denote that B is
-    // not worse than A.
     //
+    // We write this as A > B, and write !(A > B) as A <= B to denote that A is
```

Comment:
Good catch on the inequalities here and below. But why did you swap which paragraph this was in?

Reply:
Oh, that's .. actually just an artifact of the way I split the whitespace changes. If you look at the final state after the formatting change, I just wanted to offsetset and whitespace-pad the definitional equation above.